app/models/content_configuration.rb

```diff
@@ -31,6 +31,22 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
   preference :group_signup_case_studies_html, :text, default: I18n.t(:content_configuration_case_studies)
   preference :group_signup_detail_html, :text, default: I18n.t(:content_configuration_detail)
 
+  # Main URLs
+  preference :menu_1, :boolean, default: true
+  preference :menu_1_icon_name, :string, default: ""
+  preference :menu_2, :boolean, default: true
+  preference :menu_2_icon_name, :string, default: ""
+  preference :menu_3, :boolean, default: true
+  preference :menu_3_icon_name, :string, default: ""
+  preference :menu_4, :boolean, default: true
+  preference :menu_4_icon_name, :string, default: ""
+  preference :menu_5, :boolean, default: true
+  preference :menu_5_icon_name, :string, default: ""
+  preference :menu_6, :boolean, default: true
+  preference :menu_6_icon_name, :string, default: ""
+  preference :menu_7, :boolean, default: true
```

Comment:
From the other file it looks like items 4 and 5 are Aus-specific links, and items 6 and 7 have blank URLs by default. Should all of these menu item preferences default to being enabled in that case?

Reply:
So I think menu items 1, 2, 3, 4 and 7 are default for all instances, 4 and 5 are just for Aus, and 6 is different for each instance but optional.

Reply:
Also, the default menu items do have default icons already, so those can be added here under the `*_icon_name` preferences.

Reply:
thanks @Matt-Yorkley . I can do that, but I wouldn't do it.
I left the Aus stuff in en.yml because I don't think we have other option, but in terms of preferences defaults in the code, I'd leave everything empty/true. It's much cleaner, each instance can do what they want with the menu with just a few clicks afterwards.

If you think we should do it, can you please clarify exactly what defaults would you leave?
Did you suggest having menus 1, 2, 3, 4 and 7 to true, 5 and 6 to false and icons 1, 2, 3, 4 and 7 set to the respective icons of Shops, Map, Producers and Groups?

Reply:
It just makes sense to me that the defaults are set for more or less "universal" settings, and any extras need to be enabled. So for items 5 and 6 which only show Aus content by default and will not be used by most instances, it seems sensible not to enable them on all instances by default.

Reply:
I'm mostly thinking of minimising additional extra work that will have to be done on each instance here, so instance managers have an easier job. 

I would definitely add the default logos for standard menu items from the current codebase into the defaults here, otherwise every instance will need to go through the codebase to find each icon name for each of those icons and re-write it into the correct place, checking each time to see they got the numbers matched up to the correct menu item. And it would need to be re-done every time a new staging/production server is set up.

Reply:
Lets imagine a case where this gets merged and an instance manager forgets to manually update all these items... it would be good if the menus are not broken.

Reply:
Hi Matt, I think I understand you.
Please have a look at the new commit.
Now default menu is 5 elements (1 to 5) with default icons and default URLs as well (I set /groups as Groups default and About default as openfoodnetwork.org).
Let me know if this was you meant.
Thanks, Luis